polyhost/res/overlay_sources/teams/fetch_icons.py

```
@@ -0,0 +1,140 @@
+
+
+def _draw_teams_logo(path: Path) -> None:
+    """Generic, license-clean Teams program mark (no Microsoft logo). Uses the
+    same generic 'Office document' style as the Word mark — a 90deg-rotated
+    trapezoid page on the left with a letter knocked out (negative space), plus
+    a few text lines on the right — but stamped with a 'T' instead of a 'W', so
+    Teams sits visually with the other Office apps. White shape on transparent
+    (alpha = the lit shape) -> program_icon_mode: alpha."""
+    ss = 4
+    u = RENDER_PX * ss
+    px = lambda n: n * ss          # final px -> supersampled px
```

Ruff (0.15.17)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `px` as a `def`

(E731)